cli/tools/test.rs

```diff
@@ -650,7 +650,7 @@ impl PrettyTestReporter {
           failures_by_origin.entry(origin.clone()).or_default();
         let _ = uncaught_error.insert(js_error.as_ref());
       }
-      println!("\n{}\n", colors::white_bold_on_red(" ERRORS "));
+      println!("\n{}\n", colors::white_bold_on_red(" ERRORS"));
```

Comment:
I added these spaces on purpose. I believe it will show up the space before "ERRORS" to have red background, but there won't be any after the word. Maybe let's remove the space being colored?

Reply:
Oh, oops. I think I even removed these in the past by accident.